src/pages/stacks-wallet/usage.md

```diff
@@ -285,3 +285,21 @@ you do not restart the wallet, you can simple close it.
 
    The wallet displays the **Terms of Use**. You can accept to enter the wallet
    again or choose **Quit**.
+
+## Trezor wallet support
+
+Due to technical compatibility issues, we are currently unable to fully support all Trezor wallets. Going forward, support for any Trezor wallet will be discontinued.
```

Comment:
While we expect Trezor support to get discontinued upon mainnet, it's not an absolutely 100% done deal. There's a small chance we we do support it upon or soon after mainnet. So we may not want to provide such forward guidance quite yet.